Source/WebCore/platform/text/mac/TextBoundaries.mm

```diff
@@ -91,16 +91,24 @@ static CFStringTokenizerRef tokenizerForString(CFStringRef str)
 }
 
 // Simple case: A word is a stream of characters delimited by a special set of word-delimiting characters.
+template<bool is8Bit = false>
 static void findSimpleWordBoundary(StringView text, int position, int* start, int* end)
 {
     ASSERT(position >= 0);
     ASSERT(static_cast<unsigned>(position) < text.length());
 
+    auto string = (is8Bit) ? text.span8() : text;
```

Comment:
This parenthesis seems unnecessary?

Reply:
I always forget about that for some reason. Will fix.